src/components/portfolio/DetailsMainAnalytics.tsx

```diff
@@ -82,10 +111,27 @@ const DetailsMainAnalytics: React.FC<DetailsMainAnalyticsProps> = ({
       <div className={classes.titleWrapper}>
         {t('portfolio.details.analytics.volatility.vsMarket')}
       </div>
-      <DetailsVolatilityGraph
-        positions={portfolio.positions}
-        portfolioVolatility={portfolio.analytics.volatility}
-      />
+      <VolatilityGraph color={palette.primary.contrastText}>
+        <LargeVolatilityLineEntry
+          volatilityValue={portfolio.analytics.volatility}
+          name={t(
+            'portfolio.details.analytics.volatility.myPortfolio'
+          ).toString()}
+          textColor={palette.primary.contrastText}
+        />
+        {Object.entries(sortedStocks).map(([volatility, symbol]) => (
+          <VolatilityLineEntry
+            key={symbol}
+            volatilityValue={parseFloat(volatility)}
+            tooltipText={symbol}
+            color={
+              parseFloat(volatility) > 1
+                ? palette.error.main
+                : palette.success.main
```

Comment:
These are the colors of the small lines with the stock names. Are they not supposed to be colored?

Reply:
Yes, I got confused myself, since these colors had changed in a pull request I recently approved. But I just went on main and checked again, these colors were reverted to the original palette again. This should be fine, sorry for the confusion